modules/client-configuration/pages/vhm-gce.adoc

```diff
@@ -40,6 +40,46 @@ This is required for subscription matching to work.
 To see which objects and resources have been inventoried, navigate to menu:Systems[System List > Virtual Systems].
 
 
+
+== Assigning Permissions
+
+The VHM you create needs to have the correct permissions assigned, in order for it to access the GCE VM.
+
+Log in to your Google Cloud Platform account as an administrator, and ensure that the service account has the appropriate roles.
```

Comment:
I think important part is, that it is necessary to add service account to the IAM in the project. (That's exactly the part I don't know before and was a bit tricky to determine.)